public/_redirects

```diff
@@ -0,0 +1,2 @@
+/*    /index.html   200
```

Comment:
is this commented out or is that just the syntax

Reply:
that do be the syntax